cumulusci/tasks/salesforce/communities_preflights.py

```diff
@@ -0,0 +1,21 @@
+import requests
```

Comment:
What would you think about organizing tasks that are primarily meant as preflight checks (i.e. gathering info rather than mutating the org) into a cumulusci.tasks.preflight package, rather than continuing to grow the already-large cumulusci.tasks.salesforce?

Reply:
I'm in favor of the idea. I didn't do it here because I wanted to keep the scope of the PR limited (e.g., regression-testing to make sure existing tasks aren't impacted in MetaDeploy after reorganization). Would you be comfortable with creating the package here with these new tasks and doing the reorganization as a separate PR?

Reply:
Yep.

Reply:
Canceling the above; I went ahead and did the whole move, leaving behind compatibility aliases (eyes appreciated - I _think_ I hit what's needed).

Reply:
I immediately hit an import cycle (trying to import GetInstalledPackages imports BaseSalesforceApiTask from the salesforce package, which itself tries to import GetInstalledPackages), so I bit the bullet and rewrote cumulusci.tasks.salesforce to import things lazily. @davidmreed @prescod want to see if you can break it?

Reply:
You've updated it since I started reviewing this morning and addressed most of my concerns.

I guess the only remaining one is whether this mechanism should move to a Utility class to be used anywhere we need to do lazy module loading (either for backwards compatibility or just because we want shorter moodule paths)